CONTRIBUTING.md

```diff
@@ -1 +1,19 @@
-Please see our guidelines in the wiki: https://github.com/commons-app/apps-android-commons/wiki/Volunteers-welcome%21
+Thanks for considering to contribute to this project! A few guidelines for
+people who want to contribute their code to this software are documented in
+[this project's Wiki](https://github.com/commons-app/apps-android-commons/wiki/Contributing-Guidelines).
+If you're not sure where to start head on to [this wiki page](https://github.com/commons-app/apps-android-commons/wiki/Volunteers-welcome!).
+
+Here's a jist of the guidelines,
```

Comment:
Use `gist`

Ref: https://writingexplained.org/jist-or-gist-difference

Reply:
Nice catch. Thanks for the correction!